samples/buildstrategy/ko/buildstrategy_ko_cr.yaml

```diff
@@ -110,7 +110,7 @@ spec:
           # Determine the ko version
           KO_VERSION="${PARAM_KO_VERSION}"
           if [ "${KO_VERSION}" == "latest" ]; then
-            KO_VERSION=$(curl --silent "https://api.github.com/repos/google/ko/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')
+            KO_VERSION=$(curl --silent "https://api.github.com/repos/ko-build/ko/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')
```

Comment:
FYI you can add `--location` here too to follow redirects in case it moves again, but we have no plans to move again. I might update setup-ko to do this in the future, but it's not a high priority.

Reply:
Thought about it. But same thinking. I don't assume you will move again soon.